modules/bim/game/main/include/bim/game/arena.hpp

```diff
@@ -5,6 +5,7 @@
 
 #include <entt/entity/fwd.hpp>
 
+#include <bim/table_2d.hpp>
```

Comment:
Pease move this include directive above entt. I sort the include directives from most specific to more general, with a blank line between libraries:

```c++
#include <bim/game/…>

#include <bim/table_2d.hpp>

#include <entt/…>
```

I find the rationale from [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#include-style) quite clear on the topic.